src/Domain/Composer/Version/DrupalCoreVersionResolver.php

```diff
@@ -400,4 +403,20 @@ private function convertToDev(string $version): string {
     return implode('.', $parts);
   }
 
+  /**
+   * Validates if the version is "NEXT MAJOR LATEST MINOR BETA OR LATER".
```

Comment:
Is this quite right? I feel like it should be something more like "Asserts that a next major, latest minor beta-or-later Drupal core version exists."